frontend/src/views/detail/index.tsx

```diff
@@ -35,7 +40,21 @@ export const Detail: React.FC = (): React.ReactElement => {
    * @async
    */
   const handleDelete = async () => {
-    await deleteBookById(id);
+    await deleteBookById(id)
```

Comment:
I don't like this kind of syntax await + then & catch 
anyway we will talk about it in classroom lesson